src/Runner.Worker/GitHubContext.cs

```diff
@@ -23,6 +23,7 @@ public sealed class GitHubContext : DictionaryContextData, IEnvironmentContextDa
             "ref",
             "repository",
             "repository_owner",
+            "retention_days",
```

Comment:
This means a environment variable GITHUB_RETENTION_DAYS will shows up, what retention is this for? Workflow run? Log? Artifacts?

Reply:
This is "Artifacts & Logs Retention" on UI.  We will store this value on each run, so it's effectively the run retention.  This value will be consumed by the artifact toolkit to verify user do not enter a larger value.  This value is also checked on service layer.